tgui/packages/tgui/interfaces/Manufacturer/index.tsx

```diff
@@ -0,0 +1,321 @@
+                    ))}
+                  </LabeledList>
+                </Section>
+              </Stack.Item>
+              <Stack.Item>
+                <Section
+                  textAlign="center"
+                  title={"Fabricator Settings"}
+                >
+                  <LabeledList>
+                    <LabeledList.Item
+                      label={"Repeat"}
+                      buttons={<Button icon="repeat" onClick={() => toggleRepeat()}>Toggle Repeat</Button>}
+                      textAlign="center"
+                    >
+                      {data.repeat ? "On" : "Off"}
+                    </LabeledList.Item>
+                    <LabeledList.Item
+                      label="Speed"
+                    >
+                      <Slider
+                        minValue={1}
+                        value={data.speed}
+                        maxValue={data.hacked ? data.max_speed_hacked : data.max_speed_normal}
+                        step={1}
+                        stepPixelSize={100}
+                        disabled={data.mode !== "working"}
+                        onChange={(_e: any, value: number) => act("speed", { "value": value })}
+                      />
+                    </LabeledList.Item>
+                  </LabeledList>
+                </Section>
+                {data.manudrive.limit !== null ? (
+                  <Stack.Item>
+                    <Section
+                      title="Loaded Manudrive"
+                      buttons={
+                        <Button
+                          icon="eject"
+                          content="Eject"
+                          disabled={data.mode !== "ready"}
+                          onClick={() => act("manudrive", { "action": "eject" })}
+                        />
+                      }
+                      mb={1}
+                    >
+                      {data.manudrive.name}
+                      <Divider />
+                      <LabeledList>
+                        <LabeledList.Item
+                          label={"Fabrication Limit"}
+                        >
+                          {data.manudrive.limit === -1 ? "Unlimited" : `${data.manudrive.limit} ${pluralize("use", data.manudrive.limit)}`}
+                        </LabeledList.Item>
+                        {data.manudrive.limit !== -1 ? (
+                          <LabeledList.Item
+                            label={"Remaining Uses"}
+                          >
+                            {data.manudrive_uses_left}
+                          </LabeledList.Item>
+                        ) : null }
+                      </LabeledList>
+                    </Section>
+                  </Stack.Item>
+                ) : null}
+                {data.panel_open ? (
+                  <Stack.Item mb={1} >
+                    <CollapsibleWireMenu wirePanel={wirePanel} />
+                  </Stack.Item>
+                ) : null}
```

Comment:
```suggestion
                {!!data.panel_open && (
                  <Stack.Item mb={1} >
                    <CollapsibleWireMenu wirePanel={wirePanel} />
                  </Stack.Item>
                )}
```
(Assuming `panel_open` is a BooleanLike, anyway)